router/transformer/transformer.go

```diff
@@ -65,6 +65,7 @@ type ProxyRequestMetadata struct {
 type ProxyRequestPayload struct {
 	integrations.PostParametersT
 	Metadata []ProxyRequestMetadata `json:"metadata"`
+	Config   map[string]interface{} `json:"config"`
```

Comment:
DestinationConfig would be better avoid confusion

Reply:
Incorporated the changes.